<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateAcc</class>
 <widget class="QDialog" name="CreateAcc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1880</width>
    <height>986</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">*
{
	font-family: century gotic;
	font-size: 24px;
}

QComboBox
{
	font-size:17px;
	text-align: left;

}
QGroupBox
{
	font-size: 15px;
	font-weight:bold;
}
QFrame#frame
{
	background: #F8F9F9;
	border-radius: 15px;
	border-style: dashed;
	border-color: red;

}

QPushButton
{
	background-color: red;
	border-radius: 15px;
	color: white;
}
QPushButton:hover
{
	
	border-radius: 15px;
	background: red;

}


QToolButton
{
	background: white;
	border-radius: 60px;
}
QLabel
{
	font-size: 12px;
	color: black;
}
QLabel#lblWelcomeMsg
{
	font-size: 15px;
	
}

QLabel#lblWelcome
{

	color:black;
	font-weight: bold;
	font-size:20px;
}


QLabel#lblHospital_name
{

	color:red;
	font-weight: bold;
	font-size:32px;
	text-shadow:1px 1px;
}
QLabel#lbl_doctorImage
{
	background-image: url(:/image/medical team.png);
	background-size: contain;
	background-color:transparent;
}
QLineEdit
{
	background: transparent;
	border: none;
	color: #717072;
	background-color:#C8CFD1;
	border-bottom: 1px solid #717072;
}
</string>
  </property>
  <widget class="QLabel" name="lbl_doctorImage">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>130</y>
     <width>741</width>
     <height>481</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>581</width>
     <height>901</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="lblWelcome">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>391</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Thank you for being part of the Team!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>541</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QToolButton" name="toolButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/image/firstaidicon.jpg</normaloff>:/image/firstaidicon.jpg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>400</width>
       <height>400</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="lblHospital_name">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>411</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>First Aid Hospital</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lblWelcomeMsg">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Lets get you signed up</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_login">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>820</y>
      <width>461</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Create Account</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbPersonalInfo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>531</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Personal Information</string>
    </property>
    <widget class="QLineEdit" name="lineEditFirstName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lblFirstName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>First Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditLastName">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>60</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lblLastName">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>40</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Last Name</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbAccountInfo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>531</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Account Information</string>
    </property>
    <widget class="QLineEdit" name="LineEditUserName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lblUserName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>UserName</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditPassword">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>50</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lblPassword">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>30</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbAccountInformation">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>531</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Address Information</string>
    </property>
    <widget class="QLineEdit" name="lineEditStreetAddress1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lblStreetAddress1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Street Address #1</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblStreetAddress2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Street Address #2</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditStreetAddress2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>40</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditCity">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lblCity">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>City</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblState">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>90</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>State</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditZipCode">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lblZipCode">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Zip Code</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblCountry">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>160</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Country</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbStatesName">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>110</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="insertPolicy">
      <enum>QComboBox::InsertAfterCurrent</enum>
     </property>
    </widget>
    <widget class="QComboBox" name="cbCountyNames">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>180</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="insertPolicy">
      <enum>QComboBox::InsertAfterCurrent</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbContactInfo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>670</y>
      <width>531</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Contact Information</string>
    </property>
    <widget class="QLineEdit" name="lineEditEmail">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lblEmail">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Email Address</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditPhoneNumber">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>50</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lblPhoneNumber">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>30</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Phone Number</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
